server/routerlicious/packages/routerlicious-base/src/alfred/runnerFactory.ts

```diff
@@ -432,6 +433,7 @@ export class AlfredResourcesFactory implements core.IResourcesFactory<AlfredReso
 		const port = utils.normalizePort(process.env.PORT || "3000");
 
 		const deltaService = new DeltaService(operationsDbMongoManager, tenantManager);
+		const documentDeleteService = new DocumentDeleteService();
```

Comment:
1. Add a flag?
2. You will need to add a new filed in server/routerlicious/packages/routerlicious-base/src/alfred/customizations.ts so FRS could inject its own implementation

[ Sample](https://github.com/microsoft/FluidFramework/pull/15264/files#diff-0c71f4c37b508b9abd3e4a6458e3b48180091ce4662d13a4e52ed3e615f0528e)